src/WebJobs.Extensions.ApiHub/Listener/ApiHubListener.cs

```diff
@@ -41,7 +41,7 @@ public void Dispose()
 
         public Task StartAsync(CancellationToken cancellationToken)
         {
-            _poll = _folderSource.CreateNewFileWatcher(OnNewFile, 5);
+            _poll = _folderSource.CreateNewFileWatcher(OnNewFile, pollIntervalInSeconds: 15);
```

Comment:
Eventually you'll probably want to consider making the poll interval a config knob that users can set. All our other bindings make things like this configurable.